### PR TITLE
Refresh NuGet get started articles for VS 2022 #3

### PR DESCRIPTION
**Pull request by dksimpson**
Refresh *Install and manage packages in Visual Studio using the NuGet Package Manager* article.

This work is in accordance with [User Story 1973817](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1973817).

### CODE REVIEW

**Comment by dksimpson** [comment]
Hello @Mikejo5000 @nkolev92, @donnie-msft @JonDouglas @zivkan, this articles is ready for review.

**Comment by dksimpson** [comment]
@Mikejo5000 @nkolev92, @donnie-msft @JonDouglas @zivkan: One thing I've noticed is that the Options control in the VS NuGet Package Manager doesn't reveal any other options except **Show preview window**. I tried all the other packages and none of them displayed any of the additional options mentioned in the article.

**Review by Mikejo5000 — APPROVED** [review]
LG to me, thanks!

**Review by zivkan — APPROVED** [review]

File: docs/consume-packages/install-use-packages-visual-studio.md [hunk 1]
```diff
@@ -14,155 +14,183 @@ f1_keywords:
...
-The Package Manager installs the selected package version into all selected projects, after which the package no longer appears on the **Consolidate** tab.
+   The NuGet Package Manager installs the selected package version into all the selected projects, after which the package no longer appears on the **Consolidate** tab.
 
 ## Package sources
 
-To change the source from which Visual Studio obtains packages, select one from the source selector:
+To manage NuGet package sources, follow these steps:
 
-![Package source selector in the package manager UI](media/PackageSourceDropDown.png)
+1. To change the source from which Visual Studio obtains packages, select a source from the **Package source** selector.
```
Comment (donnie-msft) [inline docs/consume-packages/install-use-packages-visual-studio.md:117]: "Obtains packages" isn't quite accurate. The UI loads metadata from versions of a package based on the source selected. However, NuGet Restore doesn't care which drop-down source is selected.

> If a package isn't already installed, NuGet first attempts to retrieve it from the [cache](https://docs.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders). If the package isn't in the cache, NuGet tries to download the package from all enabled sources in the list at Tools > Options > NuGet Package Manager > Package Sources in Visual Studio.

https://docs.microsoft.com/en-us/nuget/consume-packages/package-restore#package-restore-overview
https://docs.microsoft.com/en-us/nuget/consume-packages/package-source-mapping

There is a new feature that *will do this*, called **Package Source Mapping**. See https://docs.microsoft.com/en-us/nuget/consume-packages/package-source-mapping

File: docs/consume-packages/install-use-packages-visual-studio.md [hunk 2]
```diff
@@ -14,155 +14,183 @@ f1_keywords:
...
 
-To manage package sources:
+   ![Screenshot showing the Package source selector highlighted.](media/package-source-selector.png)
 
-1. Select the **Settings** icon in the Package Manager UI outlined below or use the **Tools > Options** command and scroll to **NuGet Package Manager**:
+1. To change the package source options, select the **Settings** icon or select **Tools** > **Options**.
```
Comment (donnie-msft) [inline docs/consume-packages/install-use-packages-visual-studio.md:121]: Personally, this reads a bit oddly to me.

```suggestion
1. To manage your package sources, select the **Settings** icon or select **Tools** > **Options**.
```

File: docs/consume-packages/install-use-packages-visual-studio.md [hunk 3]
```diff
@@ -14,155 +14,183 @@ f1_keywords:
...
 
-## Package manager Options control
+1. To remove a package source, select it, and then select the **X** button.
 
-When a package is selected, the Package Manager UI displays a small, expandable **Options** control below the version selector (shown here both collapsed and expanded). Note that for some project types, only the **Show preview window** option is provided.
+   If a package source reappears after you delete it, it might be listed in a computer-level or user-level *NuGet.config* file. For the location of these files, see [Common NuGet configurations](../consume-packages/configuring-nuget-behavior.md). Remove the package source in the files by editing them manually or using the [nuget sources command](../reference/nuget-exe-CLI-reference.md).
 
-![Package manager options](media/PackageManagerUIOptions.png)
+Using the up and down arrow buttons doesn't change the priority order of the package sources. Visual Studio ignores the order of package sources, and uses the package from whichever source is the first to respond to a request. For more information, see [Restore packages](../consume-packages/package-restore.md).
```
Comment (donnie-msft) [inline docs/consume-packages/install-use-packages-visual-studio.md:141]: I removed these buttons in 17.2 Preview 1. https://github.com/NuGet/NuGet.Client/pull/4376

**Comment by donnie-msft** [comment]
> @Mikejo5000 @nkolev92, @donnie-msft @JonDouglas @zivkan: One thing I've noticed is that the Options control in the VS NuGet Package Manager doesn't reveal any other options except **Show preview window**. I tried all the other packages and none of them displayed any of the additional options mentioned in the article.

Just noticed this question. The Options control isn't generally what's being referred to by VS Options; rather, the Tools -> Options dialog, then finding the "NuGet Package Manager" page there (or, a shortcut is Tools -> NuGet Package Manager -> Package Manager Settings) is our VS Options page.

File: docs/consume-packages/install-use-packages-visual-studio.md [hunk 4]
```diff
@@ -14,155 +14,183 @@ f1_keywords:
...
 ## Find and install a package
 
-1. In **Solution Explorer**, right-click either **References**  or a project and select **Manage NuGet Packages...**.
+To find and install a NuGet package with Visual Studio, follow these steps:
 
-    ![Manage NuGet Packages menu option](media/ManagePackagesUICommand.png)
+1. Load a project in **Solution Explorer**, and then select **Project** > **Manage NuGet Packages**.
 
-1. The **Browse** tab displays packages by popularity from the currently selected source (see [package sources](#package-sources)). Search for a specific package using the search box on the upper left. Select a package from the list to display its information, which also enables the **Install** button along with a version-selection drop-down.
+   The **Nuget Package Manager** window opens.
```
[the remark this answers is not in the document]
Comment (dksimpson) [reply docs/consume-packages/install-use-packages-visual-studio.md:39]: @donnie-msft, thanks for your comments. I've addressed them in a new PR: #2871.

**Comment by dksimpson** [comment]
> > @Mikejo5000 @nkolev92, @donnie-msft @JonDouglas @zivkan: One thing I've noticed is that the Options control in the VS NuGet Package Manager doesn't reveal any other options except **Show preview window**. I tried all the other packages and none of them displayed any of the additional options mentioned in the article.
> 
> Just noticed this question. The Options control isn't generally what's being referred to by VS Options; rather, the Tools -> Options dialog, then finding the "NuGet Package Manager" page there (or, a shortcut is Tools -> NuGet Package Manager -> Package Manager Settings) is our VS Options page.

@donnie-msft  This still doesn't match what I've documented, though - NuGet Package Manager -> Package Manager Settings describe different options.

**Pull request merged**